build/media-suite_compile.sh

```diff
@@ -1241,6 +1241,28 @@ if [[ $mp4box = "y" ]] && do_vcs "https://github.com/gpac/gpac.git"; then
     do_checkIfExist
 fi
 
+_check=(SvtHevcEnc.pc libSvtHevcEnc.dll.a svt-hevc/EbApi.h
+    bin-video/Svt-Hevc/{libSvtHevcEnc.dll,SvtHevcEnc{App.exe,SimpleApp.exe}})
+if [[ $svthevc = "y" ]] && do_vcs "https://github.com/intel/SVT-HEVC.git"; then
+    do_uninstall bin-video/SVT-Hevc/libSvtHevcEnc.dll.a "${_check[@]}"
+    ORIGINAL_PKG_CFG=$PKG_CONFIG PKG_CONFIG="$LOCALDESTDIR/bin/ab-pkg-config-static.bat"
+    do_patch -p "https://pastebin.com/raw/ryHPfJEA"
+    create_build_dir
+    CFLAGS="-O2 -pipe" log cmake cmake .. -G"Ninja" \
+        -DCMAKE_INSTALL_PREFIX="$LOCALDESTDIR" -DCMAKE_BUILD_TYPE=Release \
+        -DMINGW=on
+    log build ninja
+    do_install SvtHevcEnc.pc lib/pkgconfig/
+    do_install Release/lib/* lib/
+    cd_safe ..
+    do_install Bin/Release/* bin-video/SVT-Hevc/
+    do_install Bin/Release/libSvtHevcEnc.dll{,.a} lib/
```

Comment:
`.dll` are usually installed to the /bin-(global|video|audio) directories, since they're needed at runtime, not the compilations (directly, at least, they need them through the `.dll.a` which point to them for linking).

Reply:
I see. I will modify it after I get off this plane. However this will probably be on hold since there are still problems with the program after compilation. (The end video is split vertically and is purple)

Reply:
Although it can be compiled into x265 and ffmpeg, however right now, I'm unsure if both are actually necessary or if one is just a byproduct of improper cmake files

Reply:
If the svt-* line of encoders actually become compatible with mingw, I might just add them together